.github/workflows/aws-snp-launchmeasurement.yml

```diff
@@ -0,0 +1,67 @@
+name: AWS SNP Launch Measurement
+
+on:
+  schedule:
+    - cron: '0 22 * * *'
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Install necessary tools
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y python3 python3-pip
+        sudo python3 -m pip install sev-snp-measure==0.0.5
+
+    - name: Install Nix
+      uses: cachix/install-nix-action@v20
```

Comment:
All actions should be pinned to a commit hash. Version can be specified in a comment after that so renovate can understand it.